core/src/wallet/tezos/explorers/api/TezosLikeTransactionParser.cpp

```diff
@@ -139,6 +139,8 @@ namespace ledger {
                     _transaction->gas_limit = toValue(number, false);
                 } else if (_lastKey == "storage_limit") {
```

Comment:
I suggest to put all these raw strings to `constexpr` variables somewhere. Even if these values are used just one time, I think any potential future changes will be faster with that kind of constant values.

Reply:
You would have to change each variable’s name. I think it’s fine here.